documentation/src/developer_documentation/archie_and_fuzzy.md

```diff
@@ -0,0 +1,54 @@
+# What and How?
+
+Fuzzy and Archie are two servers hosted in the ChainSafe office. Both belong to
+the Forest team.
+
+Archie and Fuzzy are accessible through a CloudFlare tunnel. Add this to your
+SSH config (`~/.ssh/config`):
+
+```
+Host archie.chainsafe.dev
+  ProxyCommand /usr/local/bin/cloudflared access ssh --hostname %h
+  User archie
+
+Host fuzzy-forest.chainsafe.dev
+  ProxyCommand /usr/local/bin/cloudflared access ssh --hostname %h
+  User fuzzy
+```
+
+You should be able to directly ssh into `archie.chainsafe.dev` and
+`fuzzy-forest.chainsafe.dev`.
+
+# Archie
+
+Hardware:
+
+```
+Motherboard:  GIGABYTE B550M
+CPU:          AMD Ryzen™ 5 5600G
+SSD:          4x SAMSUNG 870 QVO 8 TB
+RAM:          G.Skill DIMM 32 GB DDR4-3200
+```
+
+Archie is currently storing the entire Filecoin graph. In the future, this data
+will be served into the Filecoin p2p network.
+
+# Fuzzy
+
+Hardware:
+
+```
+Motherboard:  GIGABYTE B550M
+CPU:          AMD Ryzen™ 5 5600G
+SSD:          1x Seagate FireCuda 530 2 TB
+RAM:          G.Skill DIMM 64 GB DDR4-3200
+```
+
+Fuzzy is meant to run a variety of long-running tasks.
+
+## Github Action Runner
+
+Fuzzy is running this runner: https://github.com/actions/runner
+
+The instance can be inspected by running `zellij attach runner` on the server.
+The command to start the runner is `cd ~/gc_runner; ./run.sh`.
```

Comment:
What is the GH runner config exactly? How many jobs can it run in parallel? Is it some default config?

Reply:
Everything is default. The runner can run one job at a time.

Reply:
Updated the wording.

Reply:
Actually, 3 jobs on every commit push/rebase/merge.

Reply:
One runner is fine for now. Setting up more runners isn't just changing a configuration parameter.

Reply:
I see. It seems we need to have multiple users on a machine for multiple runners. I'm surprised by it, frankly. GitLab had proper parallelization of jobs already, like 7 years ago.

I'm not convinced that 1 runner will be enough, but we can try.